.github/workflows/service_test_hdfs.yml

```diff
@@ -173,3 +173,41 @@ jobs:
           OPENDAL_HDFS_ATOMIC_WRITE_DIR: /tmp/atomic_write_dir/opendal/
           OPENDAL_HDFS_NAME_NODE: default
           OPENDAL_HDFS_ENABLE_APPEND: false
+
+  hdfs-default-gcs:
```

Comment:
We are working on the test migrate, please take a look over: https://github.com/apache/opendal/pull/4140.